### ISSUE TITLE
`listOffset` prop should be 0 by default in headless svelte-select

### ISSUE THREAD

**Issue by ZerdoX-x** [rob-balfre/svelte-select#407]
I need to redefine this prop in order to style svelte-select properly.

**Comment by rob-balfre** [comment]
@ZerdoX-x not sure I follow, you can already.

**Comment by ZerdoX-x** [comment]
So current styles that are available in no-styles svelte-select are `listOffset`. Also `top`, `left`, `bottom` and `position: fixed` for list. They should be removed too, but `top` `left` and `bottom` must be replaced with css custom properties so if user (dev) wanted to use calculated values, they would be available.

**Comment by ZerdoX-x** [comment]
> @ZerdoX-x not sure I follow, you can already.

I know that I can. But headless (no-styles) means I don't need to redefine styles (because there should be nothing to redefine)

**Comment by rob-balfre** [comment]
The list's position is calculated on the fly when the user scrolls or adjusts the browser.  Can you describe the problem you're trying to solve or even better provide a REPL? Thanks

**Comment by ZerdoX-x** [comment]
> The list's position is calculated on the fly when the user scrolls or adjusts the browser.

Yep. I understand. I want svelte-select to store these values in css custom properties inlined, then applied in regular css.

> Can you describe the problem you're trying to solve?

Yes. First of all I want to style svelte-select from zero state. 
Here you can compare how it's going for me as user on v4 and v5, trying to implement same design. https://svelte.dev/repl/e1c513804f0d4417b72d755928e27cfd?version=3.48.0

Second one has much better design and has clear external parameters via css custom properties. First can't be restyled externally.

But second one still has some issues:
1) I needed to remove unwanted gap (listOffset) before styling.
2) I don't want my list to be positioned fixed. I can redefine these styles only using `!important`, which is the worst practice in css.
